src/components/Nav/NavComponent/NavSearchContainer.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faMagnifyingGlass,
+  faCalendar,
+  faL,
+} from '@fortawesome/free-solid-svg-icons';
+import Sample from './LocationDropdown';
+
+const NavSearchContainer = () => {
+  const [isClicked, setIsCLicked] = useState(false);
+
+  return (
+    <StyledNavSearchContainer>
+      <NavSearchBox>
+        <Inputs>
+          <NavLocationBox>
+            <NavIconGlass>
+              <FontAwesomeIcon icon={faMagnifyingGlass} />
+            </NavIconGlass>
+            <NavLocationInput
+              onClick={() => {
+                setIsCLicked(!isClicked);
+              }}
+            />
```

Comment:
```suggestion
            <NavLocationInput
              onClick={() => {
                setIsCLicked(!isClicked);
              }}
            type="text" placeholder="제주도"/>
```
* styled components에서 attrs로 속성 부여 할 수도 있지만, JSX에서 이렇게도 쓸 수 있습니다.

Reply:
확인했습니다.. !